pkg/container-disk/container-disk.go

```diff
@@ -335,7 +335,7 @@ func generateContainerFromVolume(vmi *v1.VirtualMachineInstance, config *virtcon
 		copyPathArg := path.Join(volumeMountDir, mountedDiskName)
 		args = []string{"--copy-path", copyPathArg}
 
-		log.Log.Object(vmi).Infof("arguments for container-disk \"%s\": --copy-path %s", name, copyPathArg)
+		log.Log.V(3).Object(vmi).Infof("arguments for container-disk \"%s\": --copy-path %s", name, copyPathArg)
```

Comment:
@Barakmor1 Let's remove this, I don't see value in this log. You will have the arguments in the Pod spec.

Reply:
+1

Reply:
Yep i think you are right